### PR TITLE
Add seeders/leechers to notifiers

### PR DESCRIPTION
**Pull request by fernandog**
still need to update search_tests and generic_provider_tests.py and feedparser_tests.py

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/search.py [hunk 1]
```diff
@@ -169,11 +169,11 @@ def snatchEpisode(result, endStatus=SNATCHED):
...
         if curEpObj.status not in Quality.DOWNLOADED:
             try:
-                notifiers.notify_snatch(curEpObj._format_pattern('%SN - %Sx%0E - %EN - %QN') + " from " + result.provider.name)
-            except:
+                notifiers.notify_snatch(curEpObj._format_pattern('%SN - %Sx%0E - %EN - %QN') + " with " + str(result.seeders) + " seeders and " + str(result.leechers) + " leechers from " + result.provider.name)
+            except Exception as e:
                 # Without this, when notification fail, it crashes the snatch thread and SR will
                 # keep snatching until notification is sent
-                logger.log(u"Failed to send snatch notification", logger.DEBUG)
+                logger.log(u"Failed to send snatch notification. Error: %r" % repr(e), logger.WARNING)
```
Comment (fernandog) [inline sickbeard/search.py]: @miigotu maybe DEBUG?

**Comment by duramato** [comment]
Nicee 
Preety usefull to know :P since there isnt a way to know them atm

**Comment by duramato** [comment]
Also , there should be an IF to have diferent messages for usenet/torrents . Is there any variable to know what search provider it is?

**Comment by fernandog** [comment]
I added if result.seeders and .... if not show old message

**Comment by duramato** [comment]
That should work , did it also fix the error?

**Comment by fernandog** [comment]
no

**Comment by fernandog** [comment]
@miigotu cant make tests works

**Comment by fernandog** [comment]
with this we can also add seeds/leechers here:
https://github.com/SickRage/SickRage/blob/develop/sickbeard/search_queue.py#L145

it's a INFO log.

**Pull request closed without merging**